keras_mxnet_ci/pr-buildspec.yml

```diff
@@ -11,7 +11,7 @@ phases:
       echo "Installing MXNet";
       apt-get update;
       pip install --upgrade pip;
-      pip install mxnet-mkl;
+      pip install mxnet-mkl --pre;
```

Comment:
why not just `mxnet`? why are we using `mxnet-mkl` here?

Reply:
mxnet use open-blas and there was a issue in open-blas causing nightly CI to fail. When we use pytest to test keras mxnet, it crashes.

Previous investigation result:
```
Root cause found using command: py.test tests/ -n0 -vs 

Error Message: 

tests/test_ms.py::test_model_saving_to_pre_created_h5py_file OpenBLAS : Program will terminate because you tried to start too many threads. 
```